third_party/glslang/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ if(${CMAKE_CXX_COMPILER_ID} MATCHES "GNU")
     add_compile_options(-Wall -Wmaybe-uninitialized -Wuninitialized -Wunused -Wunused-local-typedefs
                         -Wunused-parameter -Wunused-value  -Wunused-variable -Wunused-but-set-parameter -Wunused-but-set-variable -fno-exceptions)
     add_compile_options(-Wno-reorder)  # disable this from -Wall, since it happens all over.
-elseif(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
+elseif(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang" AND NOT MSVC)
```

Comment:
I'm curious why this is necessary. What is MSVC's `CMAKE_CXX_COMPILER_ID`?

Reply:
I don't remember for sure, but I think one of these was not accepted by the MSVC compiler.

Reply:
Sure, but why do we need the addition of `AND NOT MSVC`?
The statement `${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"` should already evaluate to `false` for MSVC.

Reply:
@bejado, you are right that change does not appear to be necessary. I have removed it, and verified that glslang still builds, so I have checked in the removal of this change. Thanks for pointing out this mistake.